src/embed/dispatch.rs

```diff
@@ -113,7 +113,11 @@ async fn do_open(conns: &mut ConnMap, req: &Request) -> Response {
         return Err("db_name must be non-empty".into());
     }
     if conns.contains_key(&req.db_name[..]) {
-        return Ok("".into());
+        return Err(format!(
```

Comment:
I think we were relying on this behavior before... like it was allowed (api wise) to create multiple repliache instances with same name.

Reply:
Where are we relying on this? There are definitely things that do not work if we allow this. (subscriptions for sure but maybe other things too)

I also think it is a bad idea but if we want this to work we can make it work.

Reply:
If you call close on one of them then both are closed is another problem.

Reply:
Hm, maybe some old sample app, or maybe just in my mental model.

TBC, this means that only one instance of `Replicache` can be constructed for each named cached. This is kind of a surprising restriction to me just from a JS API pov.

I see the bug you are talking about in `close` and I agree that if this isn't really working let's make it official.

But I won't be surprised if users trip over this and find it confusing / limiting.